src/main/js/helpers/legend.js

```diff
@@ -293,5 +300,6 @@ export {
     loadPieLegendItem,
     removeLegendItem,
     legendClickHandler,
-    legendHoverHandler
+    legendHoverHandler,
+    isSelected
```

Comment:
If you are exporting it then you might want to namespace this like other functions in this file.

Reply:
Updated here https://github.com/cerner/carbon-graphs/pull/140/commits/a5884626cdc15cd1f6c9143a66a8af7d004296f0